Python/phate/phate.py

```diff
@@ -74,6 +74,11 @@ class PHATE(BaseEstimator):
         n_pca < 20 allows neighborhoods to be calculated in
         roughly log(n_samples) time.
 
+    optimizer : {'sgd', 'smacof'}, optional (default: 'sgd')
```

Comment:
I'm not sure the policy of default prioritizing speed over optimality is better here. I would feel much better if the optimal solver was default.

Reply:
Also for the API, TSNE in sklearn uses `method=` to specify the barnes-hut vs exact optimization (https://scikit-learn.org/stable/modules/generated/sklearn.manifold.TSNE)

>_New in version 0.17:_ Approximate optimization _method via the Barnes-Hut._ [emphasis theirs]

Isomap uses `eigen_solver=` to specify the eigenvalue decomposition method.

What about `mds_solver=` for PHATE?

Reply:
I like mds_solver, good idea

Reply:
I like `mds_solver`, good idea.